### ISSUE TITLE
[BUG] Demo/Simple page radio buttons not using proper HTML input elements with type="radio".

### ISSUE THREAD

**Issue by adambuttrick** [CDLUC3/ezid#936]
**Describe the bug**
Radio button controls on the Demo/Simple page (label.fcontrol__radio-label-stacked:nth-child(2)) are not using proper HTML input elements with type="radio". This makes it harder for screen readers and assistive technologies to properly interpret and interact with these controls.

**To Reproduce**
1. Navigate to Demo/Simple page
2. Inspect the radio button controls at label.fcontrol__radio-label-stacked:nth-child(2)
3. Notice they are not using standard <input type="radio"> elements
4. Screen readers may not properly announce these as radio buttons or handle selection correctly

**Expected behavior**
Radio buttons should be implemented using standard HTML:
```html
<input type="radio" name="group" id="option1" value="option1">
<label for="option1">Option Label</label>
```
